web/client-api/src/main/java/io/deephaven/web/client/api/barrage/WebBarrageUtils.java

```diff
@@ -100,12 +100,12 @@ public static ColumnDefinition[] readColumnDefinitions(Schema schema) {
             cols[i].setStyleColumnName(fieldMetadata.get("styleColumn"));
 
             if (fieldMetadata.containsKey("inputtable.isKey")) {
-                cols[i].setInputTableKeyColumn(Boolean.parseBoolean(fieldMetadata.get("inputtable.isKey")));
+                cols[i].setInputTableKeyColumn("true".equals(fieldMetadata.get("inputtable.isKey")));
```

Comment:
Why not use `Boolean.parseBoolean`? Seems clear and also doesn't restrict to case sensitivity:
```
    public static boolean parseBoolean(String s) {
        return "true".equalsIgnoreCase(s);
    }
```

Reply:
That's fair, could rewrite all of the - I just picked what we were being consistent around and made it 100% so, instead of ~80% (with one equivalent call, and one wrong).

Reply:
Also fair